README.md

```diff
@@ -167,23 +169,45 @@ Use [`func (tf *TF) Cmd(name string, args ...string) *exec.Cmd`](https://pkg.go.
 
 ### Verbose mode
 
-Enable verbose output using the `-v` CLI flag. It works similar to `go test -v`. When enabled, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
+In verbose mode, the whole output will be streamed. If disabled, only logs from failed task are send to the output.
 
-Use [`func (*TF) Verbose`](https://pkg.go.dev/github.com/pellared/taskflow#TF.Verbose) to check if verbose mode was set within the task's command.
+By default, verbose mode is turned off.
```

Comment:
I think this is really not true.

When I commented the `taskflow.VerboseParam(flow)` in `build/build.go` then it was always running as it was verbose.

However, it is NOT verbose by default if `taskflow.VerboseParam(flow)` was called.

IMO the behavior is correct. I would just fix the docs.

Reply:
You are right, if no parameter is given, it should print all.
Plus, I got confused with the boolean logic myself :)